dts/msm8226/rules.mk

```diff
@@ -1,6 +1,7 @@
 LOCAL_DIR := $(GET_LOCAL_DIR)
 
 DTBS += \
+	$(LOCAL_DIR)/msm8926-huawei-g6-l11-vb.dtb \
```

Comment:
I think it's nicer to have this list sorted alphabetically

Reply:
@TravMurav Thanks for your review, I'm submitting another patch, keeping the grouping by SoC like for other targets.

Reply:
Unfortunately, the stock bootloader matches the asus zenwatch dtb instead when the g6-l11-vb one is after. On a previous version of this series, I added the g6-l11-va dts before the g6-l11-vb one, and the correct one was matched on a g6-l11-vb device, so the stock bootloader does not always match the first dtb, although order is important somehow.

Reply:
@mirsal So does it not work anymore at all with the latest changes you pushed here (since it isn't the first DTB anymore)?

Reply:
@stephan-gh Hi, indeed, the last changes do break it. You can probably close this PR, I'll open a new one if I get another opportunity at working on this device.

The strange thing is that the stock bootloader does not necessarily match the first dtb entry (as illustrated by the patch version with both va and vb, in which the va dtb appears first, but the vb dtb was correctly matched on a vb device) but in this case the bootloader still matches the asus-sparrow dtb as the g6-l11 dtb node appears after.

Reply:
You can add the typical `quirky - see comment in ...` comment in the README for now that usually suggests to build the lk2nd.img only with some selected DTBs instead of including all of them.

Reply:
done, I added the quirky notice and comment in a separate commit